packages/webview_flutter/android/src/main/java/io/flutter/plugins/webviewflutter/FlutterWebView.java

```diff
@@ -436,6 +453,27 @@ private void updateUserAgent(String userAgent) {
     webView.getSettings().setUserAgentString(userAgent);
   }
 
+  private void zoomEnabled(boolean shouldEnable) {
+    webView.getSettings().setSupportZoom(shouldEnable);
+  }
+
```

Comment:
Could you also add the `setTextZoom`,  as the android font will follow the font changes set by the system, the behavior is different from IOS.

Reply:
Sorry for the delay. I think that is out of scope for this PR as really, I'm just trying to be able enable/disable zoom on Android and iOS.